orbit/pkg/packaging/macos.go

```diff
@@ -124,9 +127,25 @@ func BuildPkg(opt Options) (string, error) {
 		}
 	}
 
+	cert, certOK := os.LookupEnv("MACOS_DEVID_CERTIFICATE")
```

Comment:
Should this be a `opt.MacOSDevIDCertificate`? (and define the env variable one level up, at the `fleetctl package` level)

Reply:
I tried to mimic `AC_API_ISSUER` in the sibling `NotarizeAndStaple` method for no good reason, I will change it then!